packages/xo-server/src/xapi-stats.js

```diff
@@ -1,10 +1,28 @@
-import endsWith from 'lodash/endsWith'
 import JSON5 from 'json5'
 import limitConcurrency from 'limit-concurrency-decorator'
 import { BaseError } from 'make-error'
+import {
+  endsWith,
+  findKey,
+  forEach,
+  identity,
+  isFunction,
+  isString,
```

Comment:
Use `typeof value === 'function'` and `typeof value === 'string'` instead. (as simple, more idiomatic, less dependent)